bundle/manifests/nginx-ingress-operator.clusterserviceversion.yaml

```diff
@@ -364,14 +364,20 @@ spec:
                 - --secure-listen-address=0.0.0.0:8443
                 - --upstream=http://127.0.0.1:8080/
                 - --logtostderr=true
-                - --v=10
+                - --v=0
                 image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
                 name: kube-rbac-proxy
                 ports:
                 - containerPort: 8443
                   name: https
                   protocol: TCP
-                resources: {}
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 128Mi
+                  requests:
+                    cpu: 5m
+                    memory: 64Mi
```

Comment:
These resource limits have been causing a lot of OOM issues requiring manual intervention in customer environments - we either need to remove them again or figure out a more appropriate limit (which may be quite difficult to do). If there are a lot of resources requiring processing on start up, the initial burst can overwhelm these limits quite quickly, causing an infinite loop of OOMs and pod restarts.

Reply:
I've just noticed that this is for the rbac proxy image rather than the manager image. The point still stands though - if we are doing this update, we should address the limits issue for the manager image also. Additionally, we need to test these rbac proxy limits to see if they are sufficient before making this change.

Reply:
This change is coming from the SDK, so I would trust that it was tested. We never tested the RBAC container before, I've no idea how we would go about that

Reply:
Sorry if I wasn't clear, the SDK has given us resource limits before (for the manager pod) that have caused issues in customer environments (requiring manual intervention after deployment to increase the resource limits).

I think we should create and deploy this change to a cluster and make sure we don't see OOM issues with the pod. We can't assume what is coming from the SDK has been tested sufficiently for our use case.

Finally, possibly out of scope for this PR, but the resource limits we have for the manager pod need to be either increased or removed, because testing has shown they are not sufficient. We were just going to address this as part of the Helm upgrade, but as we ARE doing a patch release now, we should address that issue as part of 0.5.1. I can create a ticket for that if required (or pop it as a subtask under the current operator release ticket).

Reply:
I don't think this is relevant for kube-rbac-proxy, our use case is not different than any other deployment.